vertical-pod-autoscaler/pkg/features/features.go

```diff
@@ -48,6 +48,15 @@ const (
 	// InPlaceOrRecreate enables the InPlaceOrRecreate update mode to be used.
 	// Requires KEP-1287 InPlacePodVerticalScaling feature-gate to be enabled on the cluster.
 	InPlaceOrRecreate featuregate.Feature = "InPlaceOrRecreate"
+
+	// alpha: v1.5.0
+	// components: recommender, updater
```

Comment:
```suggestion
	// components: admission-controller, recommender, updater
```

Since the admission-controller is performing the validation?